Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -169,6 +169,30 @@ QPolygonF GetPolygonFromRadiusGetter(F&& radius_getter)
   return shape;
 }
 
+// Constructs a polygon by querying a radius at varying angles:
+template <typename F>
+QPolygonF GetPolygonSegmentFromRadiusGetter(F&& radius_getter, double direction, double size)
```

Comment:
I'm being picky, but I don't like the look this adds to the indicators. It's very.. invasive.
I think I'd rather have something more subtle, maybe a little arc drawn in the 8 corners? (this might make more sense with my comment about using `MathUtil::Lerp` to adjust the angle).
I'd be fine merging it appearance-wise as-is though to fix it later.